app/models/html_attachment.rb

```diff
@@ -74,7 +74,11 @@ def url(options = {})
     # composed of document type, slug for the parent document, and identifier for the attachment;
     # for non-english attachments the identifier is the content_id, for english
     # attachments it is self i.e. the slug
-    identifier = sluggable_locale? ? self : content_id
+    identifier = if attachable.updated_at < Time.zone.local(2021, 11, 29)
```

Comment:
Worth adding a time in here, so we can time the deploy. E.g. set the new behaviour to apply from 7am, and arrange to do the deploy in the 30 minutes before that. We'll need to have a think about what time works best (I suspect we'll need to arrange an out-of-hours deployment to minimise risk here).

This code could also do with a big comment above it, explaining how this is a temporary measure while we patch up the old editions, and linking to that second Trello card.

Reply:
I wondered about doing the deploy in-hours on a day before the configured time, and then letting it take effect at midnight — but also there's nothing to lose by setting a time as well. (Though I assume it's unlikely that there'd be much publishing going on overnight anyway.)